# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: maciej.suminski@cern.ch-20140113121011-j135z3s0k5rn03u8
# target_branch: lp:kicad
# testament_sha1: 92c701dd98208597302634e90959423ee3de4461
# timestamp: 2014-01-13 13:10:22 +0100
# base_revision_id: jp.charras@wanadoo.fr-20140110201236-\
#   6pkzwq1imkntzwqe
# 
# Begin patch
=== modified file 'CMakeLists.txt'
--- CMakeLists.txt	2014-01-08 14:47:39 +0000
+++ CMakeLists.txt	2014-01-13 12:10:11 +0000
@@ -325,6 +325,8 @@
         message( FATAL_ERROR "Boost 1.54+ libraries are required." )
     endif()
 
+    add_custom_target( boost )      # it is required to meet some further dependencies
+
     message( WARNING "
 WARNING: You decided to skip building boost library.
 KiCad developers strongly advise you to build the bundled boost library, as it is known to work with KiCad.

